From: "2329574154@qq.com" <2329574154@qq.com>
Date: Thu, 17 Apr 2025 11:12:41 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 100.0% (136 of 136 strings)

Translation: openkylin-nile-new/kylin-os-manager-fault-diagnosis
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager-fault-diagnosis/vi/
---
 .../translations/fault-diagnosis_vi.ts             | 56 +++++++++++-----------
 1 file changed, 28 insertions(+), 28 deletions(-)

diff --git a/plugins/fault_diagnosis/translations/fault-diagnosis_vi.ts b/plugins/fault_diagnosis/translations/fault-diagnosis_vi.ts
index 5f1c82f..a03fbd4 100644
--- a/plugins/fault_diagnosis/translations/fault-diagnosis_vi.ts
+++ b/plugins/fault_diagnosis/translations/fault-diagnosis_vi.ts
@@ -11,7 +11,7 @@
     <message>
         <location filename="../3rdparty/internal_network_diagnosis/config_win.ui" line="264"/>
         <source>SetInner</source>
-        <translation>Kích hoạt kiểm tra nội mạng</translation>
+        <translation>Kích hoạt kiểm tra mạng nội bộ</translation>
     </message>
     <message>
         <location filename="../3rdparty/internal_network_diagnosis/config_win.ui" line="429"/>
@@ -29,7 +29,7 @@
         <location filename="../3rdparty/internal_network_diagnosis/config_win.cpp" line="159"/>
         <location filename="../3rdparty/internal_network_diagnosis/config_win.cpp" line="172"/>
         <source>IntraNetSet</source>
-        <translation>Cài đặt kiểm tra nội mạng</translation>
+        <translation>Cài đặt kiểm tra mạng nội bộ</translation>
     </message>
 </context>
 <context>
@@ -93,7 +93,7 @@
     <message>
         <location filename="../component/user_experience_dialog.cpp" line="66"/>
         <source>Fixed</source>
-        <translation>Đã được khắc phục</translation>
+        <translation>Đã khắc phục</translation>
     </message>
 </context>
 <context>
@@ -102,7 +102,7 @@
         <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="66"/>
         <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="102"/>
         <source>Help</source>
-        <translation>Giúp đỡ</translation>
+        <translation>trợ giúp</translation>
     </message>
     <message>
         <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="68"/>
@@ -131,12 +131,12 @@
     <message>
         <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="135"/>
         <source>Network-check-tool is a software that can quickly detect,diagnose,and optimize networks.</source>
-        <translation>Công cụ kiểm tra mạng là phần mềm giúp phát hiện, chẩn đoán và tối ưu hóa mạng một cách nhanh chóng, với quy trình thao tác đơn giản và tiện lợi.</translation>
+        <translation>Công cụ kiểm tra mạng là một phần mềm giúp kiểm tra, chẩn đoán và tối ưu hóa mạng một cách nhanh chóng, với quy trình thao tác đơn giản và tiện lợi.</translation>
     </message>
     <message>
         <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="257"/>
         <source>Network-check-tool is a software that can quickly detect, diagnose, and optimize networks. </source>
-        <translation>Công cụ kiểm tra mạng là phần mềm giúp phát hiện, chẩn đoán và tối ưu hóa mạng một cách nhanh chóng, với quy trình thao tác đơn giản và tiện lợi. </translation>
+        <translation>Công cụ kiểm tra mạng là một phần mềm giúp kiểm tra, chẩn đoán và tối ưu hóa mạng một cách nhanh chóng, với quy trình thao tác đơn giản và tiện lợi. </translation>
     </message>
     <message>
         <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="269"/>
@@ -276,7 +276,7 @@
     <message>
         <location filename="../diagnosis_subentry_widget.cpp" line="104"/>
         <source>Repair failed</source>
-        <translation>Khắc phục thất bại</translation>
+        <translation>khắc phục thất bại</translation>
     </message>
 </context>
 <context>
@@ -284,7 +284,7 @@
     <message>
         <location filename="../error_entry_widget.cpp" line="32"/>
         <source>Manual repair</source>
-        <translation>Khắc phục thủ công</translation>
+        <translation>Sửa chữa thủ công</translation>
     </message>
     <message>
         <location filename="../error_entry_widget.cpp" line="122"/>
@@ -304,12 +304,12 @@
     <message>
         <location filename="../error_entry_widget.cpp" line="199"/>
         <source>We currently do not support fixing this issue. Stay tuned!</source>
-        <translation>Tạm thời chưa hỗ trợ khắc phục vấn đề này, vui lòng chờ đợi!</translation>
+        <translation>Tạm thời chưa hỗ trợ sửa lỗi này, vui lòng chờ đợi!</translation>
     </message>
     <message>
         <location filename="../error_entry_widget.cpp" line="207"/>
         <source>Fail to create solution.</source>
-        <translation>Lỗi tạo giải pháp thất bại.</translation>
+        <translation>Tạo giải pháp thất bại.</translation>
     </message>
 </context>
 <context>
@@ -322,12 +322,12 @@
     <message>
         <location filename="../fault_diagnosis.cpp" line="148"/>
         <source>Currently running, please try again after finishing.</source>
-        <translation>Đang trong trạng thái hoạt động ,xin vui lòng thử lại sau khi kết thúc quá trình.</translation>
+        <translation>Hiện tại đang trong trạng thái chạy, vui lòng thử lại sau khi kết thúc.</translation>
     </message>
     <message>
         <location filename="../fault_diagnosis.cpp" line="204"/>
         <source>The module is not supported</source>
-        <translation>Thành phần khắc phục không được hỗ trợ</translation>
+        <translation>Thành phần sửa chữa không được hỗ trợ</translation>
     </message>
     <message>
         <location filename="../fault_diagnosis.cpp" line="220"/>
@@ -345,12 +345,12 @@
     <message>
         <location filename="../home_page_widget.cpp" line="84"/>
         <source>Quick detection and repair of fault issues, safeguarding the health of your computer</source>
-        <translation>Kiểm tra và sửa chữa sự cố nhanh chóng,duy trì hoạt động máy tính ổn định</translation>
+        <translation>Kiểm tra và sửa chữa vấn đề nhanh chóng, bảo vệ máy tính của bạn một cách toàn diện</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="132"/>
         <source>Network detection</source>
-        <translation>Kiểm tra mạng lưới</translation>
+        <translation>Kiểm tra kết nối mạng</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="91"/>
@@ -361,7 +361,7 @@
     <message>
         <location filename="../home_page_widget.cpp" line="133"/>
         <source>Troubleshooting issues with inablilty to access the internet</source>
-        <translation>Kiểm tra sự cố không thể kết nối mạng</translation>
+        <translation>Kiểm tra và loại trừ vấn đề không thể lên mạng</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="144"/>
@@ -381,17 +381,17 @@
     <message>
         <location filename="../home_page_widget.cpp" line="139"/>
         <source>Resolve hardware health and system performance issues</source>
-        <translation>Khắc phục vấn đề sức khỏe phần cứng và hiệu xuất hệ thống</translation>
+        <translation>Giải quyết các vấn đề về sức khỏe phần cứng và hiệu xuất hệ thống</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="150"/>
         <source>Online system update detection</source>
-        <translation>Kiểm tra cập nhật trực tuyến hệ thống</translation>
+        <translation>Hệ thống kiểm tra cập nhật trực tuyến</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="151"/>
         <source>Resolve issues encountered during updates and upgrades</source>
-        <translation>Khắc phục vấn đề khi cập nhật</translation>
+        <translation>Giải quyết các vấn đề gặp phải khi cập nhật</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="156"/>
@@ -401,17 +401,17 @@
     <message>
         <location filename="../home_page_widget.cpp" line="157"/>
         <source>Resolve issues encountered in software stores</source>
-        <translation>Khắc phục sự cố kho ứng dụng</translation>
+        <translation>giải quyết sự cố kho ứng dụng</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="162"/>
         <source>System disk availability detection</source>
-        <translation>Kiểm tra tính khả dụng ổ cứng hệ thống</translation>
+        <translation>Kiểm tra tính khả dụng của đĩa hệ thống</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="163"/>
         <source>Check the health status of the disk</source>
-        <translation>Kiểm tra tình trạng sức khỏe của ổ cứng</translation>
+        <translation>Kiểm tra tình trạng sức khỏe ổ đĩa</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="168"/>
@@ -460,13 +460,13 @@
     <message>
         <location filename="../mainwindow.cpp" line="328"/>
         <source>No issues fount</source>
-        <translation>Không phát hiện vấn đề</translation>
+        <translation>Chưa phát hiện vấn đề</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="317"/>
         <location filename="../mainwindow.cpp" line="329"/>
         <source>Rediagnosis</source>
-        <translation>Chẩn đoán lại</translation>
+        <translation>Kiểm tra lại</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="238"/>
@@ -487,12 +487,12 @@
     <message>
         <location filename="../mainwindow.cpp" line="309"/>
         <source>Detection interruption, no issues found</source>
-        <translation>Kiểm tra bị gián đoạn, không phát hiện vấn đề</translation>
+        <translation>Kiểm tra bị gián đoạn, chưa phát hiện vấn đề</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="312"/>
         <source>Detected interruption and found %1 issues</source>
-        <translation>Kiểm tra gián đoạn ,phát hiện %1 vấn đề</translation>
+        <translation>Kiểm tra bị gián đoạn ,phát hiện %1 vấn đề</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="334"/>
@@ -533,7 +533,7 @@
     <message>
         <location filename="../mainwindow.cpp" line="425"/>
         <source>%1 items have been tested, no issues found</source>
-        <translation>Đã kiểm tra %1 mục ,không phát hiện vấn đề</translation>
+        <translation>Đã kiểm tra %1 mục ,chưa phát hiện vấn đề</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="427"/>
@@ -548,12 +548,12 @@
     <message>
         <location filename="../mainwindow.cpp" line="436"/>
         <source>Please repair and retest</source>
-        <translation>Vui lòng kiểm tra lại sau khi khắc phục sự cố</translation>
+        <translation>Sau khi sửa chữa, vui lòng kiểm tra lại</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="441"/>
         <source>Please continue to maintain and regularly undergo physical examinations</source>
-        <translation>Vui lòng tiếp tục duy trì và thường xuyên khám sức khỏe</translation>
+        <translation>Vui lòng tiếp tục duy trì và kiểm tra định kỳ</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="443"/>
